src/main/resources/reference.conf

```diff
@@ -356,6 +365,32 @@ resourceTypes = {
     }
     reuseIds = false
   }
+  kubernetes-app = {
```

Comment:
Commented in the design doc as well, but I'm thinking we can:
* drop the `user` role -- there's no distinction between app creator and user; only the creator can use the app.
* remove the `stop` and `start` actions -- Leo won't expose this functionality for k8s
* add a `connect` action -- only the app creator should be able to connect to the app, whereas the project owner can delete but not connect.

Let me know if I'm misunderstanding something.

Reply:
where is the design doc?

Reply:
https://broadworkbench.atlassian.net/wiki/spaces/IA/pages/595492869/2020-07-07+Kubernetes+App+Sam+Model